airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -149,6 +149,7 @@ def get_dag_runs(
     execution_date_lte: Optional[str] = None,
     end_date_gte: Optional[str] = None,
     end_date_lte: Optional[str] = None,
+    state: Optional[str] = None,
```

Comment:
```suggestion
    states: Optional[List[str]] = None,
```

cc @ephraimbuddy

Reply:
We have `state` in open API doc instead of `states`, I don't think it'll work to change it to `states`